manifest.xml

```diff
@@ -8,6 +8,7 @@
     <tutorial title="Install" ref='install'>
       <markdown version="2.0">install/tutorial_2-0.md</markdown>
       <markdown version="6.x">install/tutorial_6-x.md</markdown>
+      <markdown version="7.x">install/tutorial_7-x.md</markdown>
```

Comment:
I don't think we've ever released `7.x` of libsdformat. Did you mean 9 (based on the conda command)?